operator/duties/committee.go

```diff
@@ -50,6 +50,11 @@ func (h *CommitteeHandler) HandleDuties(ctx context.Context) {
 			buildStr := fmt.Sprintf("p%v-e%v-s%v-#%v", period, epoch, slot, slot%32+1)
 			h.logger.Debug("🛠 ticker event", zap.String("period_epoch_slot_pos", buildStr))
 
+			if !h.AlanForked(slot) {
+				h.logger.Debug("🛠 skip duty execution", zap.String("reason", "alan not forked yet"))
```

Comment:
this will log every slot, I think it might be an overkill, but leave it in until you finish testing at least.